core/src/main/scala/io/finch/EncodeStream.scala

```diff
@@ -6,21 +6,21 @@ import java.nio.charset.Charset
 /**
  * A type-class that defines encoding of a stream in a shape of `S[F[_], A]` to Finagle's [[Reader]].
  */
-trait EncodeStream[S[_[_], _], F[_], A] {
+trait EncodeStream[F[_], S[_[_], _], A] {
```

Comment:
Does this improve type inference?

Reply:
It's mostly for consistency. Our other types take `F[_]` (effect) as a first argument.

Reply:
Ok, I think it probably won't matter for type inference that much. At least not in the same way that `EitherT[F, E, A]` matters. Users would probably rarely work directly with `EncodeStream`.